components/pricing.tsx

```diff
@@ -45,28 +123,38 @@ const PricingTier: React.FC<PricingTierProps> = ({
           </p>
         )}
         {isFree && (
-          <p className="text-sm font-medium text-gray-400 sm:text-base">
-            <a
-              href="https://forms.gle/171UyimgQJhEJbhU7"
-              className="text-link"
-              target="_blank"
-            >
-              Join the waitlist
-            </a>{" "}
-            to be notified when the app is available!
-          </p>
+          <>
+            <p className="text-sm font-medium text-gray-400 sm:text-base">
+              <a
+                href="https://forms.gle/171UyimgQJhEJbhU7"
+                className="text-link"
+                target="_blank"
+              >
+                Join the waitlist
+              </a>{" "}
+              to be notified when the app is available!
+            </p>
+            {!user && (
+              <p>
+                Please{" "}
+                <a href="/signin" className="text-link">
+                  log in
+                </a>{" "}
+                to download for free.
+              </p>
+            )}
+          </>
         )}
         {isFree ? (
-          ["Windows", "macOS"].map((os) => (
-            <Button
-              key={os}
-              disabled={true}
-              className="w-full rounded-2xl"
-              aria-label={`Download for ${os}`}
-            >
-              <Download className="mr-2 h-4 w-4" aria-hidden="true" /> {os}
-            </Button>
-          ))
+          numberOfDownloads < 3 ? (
+            <>
+              <DownloadButton os={getOS()} />
+            </>
+          ) : (
+            <p className="text-sm font-medium text-gray-400 sm:text-base">
+              Thank you for downloading PearAI!
+            </p>
+          )
```

Comment:
if numberOfDownloads is greater than 3, and user comes few days later to download the update, does he see download button or not? 
right now we don't push updates directly, so user have to download it from here again.